### PR TITLE
Add "avoiding disjointed features syndrome"

### PR DESCRIPTION
**Pull request by RaphHaddad**

### CODE REVIEW

**Comment by MacMoeun** [comment]
I would've like to see you expand (just a little) **on the how they
interact with the entire product** . You mentioned the **the removal of unnecessary features** which is definitely one part of it. In our previous discussions and something I agree need to be a common practice, is setting a vision or goal is important to review our upcoming features, i.e. does it align to these goal, otherwise, in your blog example, you someone might ask for a chair when our goal was a table. 

Otherwise; the rest is nice and succinct. Good blog.

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+a product. This is where useability and experience testing with early adopters
+pays off.
+
+## Pursuing the "Extensive Detail" Dichotomy
+
+When the "disjointed features" syndrome occurs, user stories will often get
+blamed for not having enough detail. Solutions that are often recommended
+during post-mortems or retrospectives is to make user stories more detailed.
+Unfortunately, this creates a situation of localised thinking within a user
+story, and more often than not, create situations where the "disjoined feature
```
Comment (CurtisLusmore) [inline content/post/avoiding-disjointed-features-syndrome.md:65]:
```suggestion
story, and more often than not, creates situations where the "disjointed feature
```

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+The immediate delivery team of a product is constantly exposed to the product,
+they understand it intimately, and are accustomed to its flaws and features.
+Often, it is a new and different perspective that highlight obvious flaws in
+a product. This is where useability and experience testing with early adopters
+pays off.
+
+## Pursuing the "Extensive Detail" Dichotomy
+
+When the "disjointed features" syndrome occurs, user stories will often get
```
Comment (CurtisLusmore) [inline content/post/avoiding-disjointed-features-syndrome.md:61]:
```suggestion
When the "disjointed features syndrome" occurs, user stories will often get
```

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+blamed for not having enough detail. Solutions that are often recommended
+during post-mortems or retrospectives is to make user stories more detailed.
+Unfortunately, this creates a situation of localised thinking within a user
+story, and more often than not, create situations where the "disjoined feature
+syndrome" is amplified rather than reduced.
+
+Localised thinking is the antithesis of systems thinking, and systems thinking
+is an important aspect in hedging against the "Disjointed Features Syndrome".
```
Comment (CurtisLusmore) [inline content/post/avoiding-disjointed-features-syndrome.md:69]:
```suggestion
is an important aspect in hedging against the "disjointed features syndrome".
```

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+![](/images/table.png)
+
+Although the table above satisfies all the requirements, it's still an
+unusable product. Another example is a disjointed document where paragraphs
+aren't orthographically consistent; such as the usage of different tense or
+points of view (among other things).
+
+There are several reason why the disjointed features syndrome may occur,
+below is an outline of a few delivery behaviours to avoid to help
```
[the remark this answers is not in the document]
Comment (RaphHaddad) [reply content/post/avoiding-disjointed-features-syndrome.md:26]: will fix.

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+in the last iteration itself, instead of the entire product end-to-end.
+
+This creates a situation where the focus is on new features, not how they
+interact with the entire product. More often than not, it isn't new features
+that make a product valuable, rather, it's the removal of unnecessary features
+that enable this visibility of value in a focussed fashion.
+
+To avoid this behaviour, beware of only reviewing the previous iteration's
+changes during the review. Whilst recent changes should be the emphasis, a
+review on the entire product, or at least, [a vertical slice](https://en.wikipedia.org/wiki/Vertical_slice) of the product each and every review will hedge against the "disjointed features syndrome".
```
[the remark this answers is not in the document]
Comment (RaphHaddad) [reply content/post/avoiding-disjointed-features-syndrome.md:42]: Will fix. Just for you Curtis!

File: content/post/avoiding-disjointed-features-syndrome.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Being agile isn't about perfectionism, it is about testing what needs to be
+perfect, and discarding things that shouldn't be pursued to perfection.
+Part of this mentality is exposing end users to the product being built early
+on in the development process. Finding friendly early adopters or testers will
+enable a systems-level perspective on the product in a way that the delivery
+team may not be able to.
+
+The immediate delivery team of a product is constantly exposed to the product,
+they understand it intimately, and are accustomed to its flaws and features.
+Often, it is a new and different perspective that highlight obvious flaws in
```
[the remark this answers is not in the document]
Comment (RaphHaddad) [reply content/post/avoiding-disjointed-features-syndrome.md:55]: Won't fix, highlight is correct as it is in the present tense.

[the remark this answers is not in the document]
Comment (handyjellyfish) [reply content/post/avoiding-disjointed-features-syndrome.md:57]: Singular present tense for highlight is highlights. 

There's only one perspective. Yes Curtis has requested backup. Don't @ me.

[the remark this answers is not in the document]
Comment (handyjellyfish) [reply content/post/avoiding-disjointed-features-syndrome.md:57]: You can either choose to write it "Often new and different perspectives highlight obvious flaws in blah blah blah" or use Curtis' fix.

[the remark this answers is not in the document]
Comment (RaphHaddad) [reply content/post/avoiding-disjointed-features-syndrome.md:57]: OK, will fix :P

**Pull request merged**